content/lua/concepts/tables/tables.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: 'Tables' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Tables are used to store collections of data.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+  - 'Lua'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Tables'
+  - 'Arrays'
+  - 'Libraries'
+  - 'Methods'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'lua/tables'
+---
+
+# **Tables**
+
+This library is internal, and will automatically work when referencing the **global** keyword **table**. In Lua, **Tables** refer to **Arrays**. <!-- Arrays HREF -->
+
+## **Functions**
+
+Below are the list of Table methods available.
+
+### table.clear(table : **table**) : void
+Clears all Indexes and Values from a table.
+
+### table.clone(table : **table**) : table
+Returns a shallow copy of the provided table
```

Comment:
```suggestion
Returns a shallow copy of the provided table.
```

Missing an ending period. You might also want to elaborate a bit on what a shallow copy means here.